scripts/examples.js

```diff
@@ -35,8 +35,9 @@ const updateRoutes = () => {
 		const route = `${snakeCase(package).toUpperCase()}_EXAMPLES_ROUTES`;
 		const moduleName = `${upperFirst(camelCase(package))}ExamplesModule`;
 
-		importConfigModules += `import { ${moduleName} } from '@acpaas-ui/ngx-examples/${package}';\n`;
-		importConfigRoutes += `import { ${route} } from '@acpaas-ui/ngx-examples/${package}';\n`;
+		// TODO: investigate this further, importing from the fesm module seems to solve the path resolving issue but should not be required to make this work
```

Comment:
is there a corresponding issue created or is this still on your todo list? otherwise i would drop the TODO because nobody is going to look at it

Reply:
i'd leave the TODO's in to keep track of these issues, this way when someone is working on the setup (which doesn't always happen in the context of an issue) and feels like cleaning this up, it doesn't get forgotten
I could make an issue as well, but it would probably be open for a long time

Reply:
ok, then just leave it in without an issue .... though in practice i think you'll be the only one to circle back and fix it